### PR TITLE
Added support for Android devices to Save to Gallery

### LINKED ISSUE #7: Implement ScreenRecordingOptions.SaveToGallery for Android
SaveToGallery is only implemented for iOS right now, can we implement an equivalent for Android?

### PR DESCRIPTION
**Pull request by sact1909**
## What's New?
- Were added support to save recorded videos to the gallery in Android Devices

## Related Issue
- Closes #7 

## Visuals

https://github.com/jfversluis/Plugin.Maui.ScreenRecording/assets/23615129/113cabf9-5fe3-4168-b9a4-569ce5edacbb

### CODE REVIEW

**Comment by jfversluis** [comment]
Oooh this is easier than I thought, thank you so much! I will have to give the code a little closer look. Will do that soon! Thank you!

**Comment by sact1909** [comment]
> Oooh this is easier than I thought, thank you so much! I will have to give the code a little closer look. Will do that soon! Thank you!

@jfversluis  yes, was very easy, but efficient 😄, I hope you like what I did when you look at the code more closely, I will wait for you to accept this one PR to start another one that solves an Issue that I opened (#14 ) , it is just a line of code but I will wait for it anyway in case you want me to change anything in this one, and of course, I will start to write about this nuget on my blog and maybe I'll do some youtube videos to show how it works, hope you agree...

File: src/Plugin.Maui.ScreenRecording/ScreenRecording.android.cs [hunk 1]
```diff
@@ -28,6 +28,8 @@ public partial class ScreenRecordingImplementation : MediaProjection.Callback, I
 
 	public bool IsSupported => ProjectionManager is not null;
 
+	public bool IsSavingToGallery { get; private set; }
```
Comment (jfversluis) [inline src/Plugin.Maui.ScreenRecording/ScreenRecording.android.cs]: I don't think this needs to be public? 
```suggestion
	bool IsSavingToGallery { get; private set; }
```

**Review by jfversluis — APPROVED** [review]
Thank you so much for this! This plugin is getting better and better!

File: src/Plugin.Maui.ScreenRecording/ScreenRecording.android.cs [hunk 2]
```diff
@@ -28,6 +28,8 @@ public partial class ScreenRecordingImplementation : MediaProjection.Callback, I
 
 	public bool IsSupported => ProjectionManager is not null;
 
+	bool IsSavingToGallery { get; private set; }
```
Comment (jfversluis) [inline src/Plugin.Maui.ScreenRecording/ScreenRecording.android.cs]:
```suggestion
	bool IsSavingToGallery;
```

**Comment by sact1909** [comment]
> Thank you so much for this! This plugin is getting better and better!

thank you man... I won't stop here, some more are coming

**Pull request merged**